### ISSUE TITLE
Example in README throws `ArgumentError` when run in docker container

### ISSUE THREAD

**Issue by piyush-kgp** [malmaud/TensorFlow.jl#497]
Sorry to be asking such a naive question, but I was trying to run the logistic reg. example in readme, and  the line `using Printf` throws `ERROR: ArgumentError: Module Printf not found in current path`. 
I am trying to run the example inside the `malmaud/julia:tf` docker container. So, maybe you could update the docker image?

**Comment by malmaud** [comment]
I think you're right; the docker image needs to be updated.

**Comment by oxinabox** [comment]
Is the docker image useful?

**Comment by piyush-kgp** [comment]
> Is the docker image useful?

yes, incredibly; for people like me, who want to quickly run the examples without having to install everything.

**Comment by oxinabox** [comment]
Cool. Good to know.
